pkg/util/pretty/spinner_text.go

```diff
@@ -9,9 +9,10 @@ import (
 // SpinnerT text style.
 //
 // Usage:
-//   sp, _ := pretty.SpinnerT.Start("Starting ...")
-//   time.Sleep(time.Second * 3)
-//   sp.Success("Done")
+//
```

Comment:
double check: https://pkg.go.dev/kusionstack.io/kusion@v0.7.2/pkg/util/pretty

![image](https://user-images.githubusercontent.com/23924124/206347624-48184647-fe80-4306-b7ab-e68548d7deed.png)

Reply:
Same format issue. It is recognized as blocker format issue in golangci-lint v1.50.1.